src/service/viewport-impl.js

```diff
@@ -602,18 +602,18 @@ export class Viewport {
    */
   animateFixedElements_(duration, curve) {
     this.fixedLayer_.updatePaddingTop(this.paddingTop_);
-    if (duration > 0) {
-      // Add transit effect on position fixed element
-      /** @const {!TransitionDef<number>} */
-      const tr = numeric(this.lastPaddingTop_ - this.paddingTop_, 0);
-      return Animation.animate(this.ampdoc.getRootNode(), time => {
-        const p = tr(time);
-        this.fixedLayer_.transformMutate(`translateY(${p}px)`);
-      }, duration, curve).thenAlways(() => {
-        this.fixedLayer_.transformMutate(null);
-      });
+    if (duration <= 0) {
```

Comment:
So, I took a look at the uses above and according to the `const duration = event['duration'];`, you actually can pass `undefined` in this method, in which case `duration <= 0` will absolutely not work as expected here - it will actually try to run an animation below with the duration of `undefined`. Probably the best course of action is to default the `duration` to 0. E.g.

```
const duration = event['duration'] || 0;
```